### PR TITLE
[DOCS]update plan only execution mode doc

### PR DESCRIPTION
**Pull request by CauchyHu**
### _Why are the changes needed?_

        it seems Flink engine is also supported

_Originally posted in https://github.com/apache/incubator-kyuubi/pull/3557#discussion_r984316871_

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate
image

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by cxzl25 — APPROVED** [review]

File: docs/client/advanced/features/plan_only.md [hunk 1]
```diff
@@ -17,16 +17,16 @@
 
 # Plan Only Execution Mode
 
-Plan only execution mode is implemented through Spark extensions, so it is only supported Spark engine so far.
+Plan only execution mode currently only supports Spark and Flink engine.
 
-It can print Spark's parsing, analysis or optimization phases execution planning, and supports application-level and session-level parameter settings, which are used in the following ways.
+Configure the kyuubi.operation.plan.only.mode parameter, The value can be 'parse', 'analyze', 'optimize', 'optimize_with_stats', 'physical', 'execution', or 'none', when it is 'none', indicate to the statement will be fully executed, otherwise only way without executing the query. different engines currently support different modes, the Spark engine supports all modes, and the Flink engine supports 'parse', 'physical', and 'execution', other engines do not support planOnly currently, and supports application-level and session-level parameter settings, which are used in the following ways.
```
Comment (ulysses-you) [inline docs/client/advanced/features/plan_only.md:22]: please revise the uppercase and lowercase

**Comment by CauchyHu** [comment]
@ulysses-you  There is a job in workflow that failed to run, is there something wrong with it? Can you retrigger it?

**Comment by ulysses-you** [comment]
@CauchyHu  do not worry, it is a flaky test

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**